library/qs_source/src/std/unstable_table_lookup.qs

```diff
@@ -274,5 +274,5 @@ namespace Microsoft.Quantum.Unstable.TableLookup {
         }
     }
 
-    export Select, WriteMemoryContents, Unlookup, MustBeFixed, EncodeUnary;
+    export Select, WriteMemoryContents, Unlookup;
```

Comment:
I mistyped on this one, both `WriteMemoryContents` and `Unlookup` are meant to be internal, so only `Select` needs to be exported.
```suggestion
    export Select;
```